Cargo.toml

```diff
@@ -40,3 +39,225 @@ members = [
+pallet-uniques ={branch = "asset-tx-payment-from-cumulus", git = "https://github.com/georgesdib/substrate/" }
+pallet-vesting ={branch = "asset-tx-payment-from-cumulus", git = "https://github.com/georgesdib/substrate/" }
+node-testing ={branch = "asset-tx-payment-from-cumulus", git = "https://github.com/georgesdib/substrate/" }
+node-executor ={branch = "asset-tx-payment-from-cumulus", git = "https://github.com/georgesdib/substrate/" }
+node-cli ={branch = "asset-tx-payment-from-cumulus", git = "https://github.com/georgesdib/substrate/" }
+node-inspect ={branch = "asset-tx-payment-from-cumulus", git = "https://github.com/georgesdib/substrate/" }
+node-rpc ={branch = "asset-tx-payment-from-cumulus", git = "https://github.com/georgesdib/substrate/" }
+pallet-contracts-rpc ={branch = "asset-tx-payment-from-cumulus", git = "https://github.com/georgesdib/substrate/" }
+pallet-mmr-rpc ={branch = "asset-tx-payment-from-cumulus", git = "https://github.com/georgesdib/substrate/" }
+sc-consensus-babe ={branch = "asset-tx-payment-from-cumulus", git = "https://github.com/georgesdib/substrate/" }
+sc-consensus-epochs ={branch = "asset-tx-payment-from-cumulus", git = "https://github.com/georgesdib/substrate/" }
+sc-consensus-babe-rpc ={branch = "asset-tx-payment-from-cumulus", git = "https://github.com/georgesdib/substrate/" }
+sc-finality-grandpa-rpc ={branch = "asset-tx-payment-from-cumulus", git = "https://github.com/georgesdib/substrate/" }
+sc-sync-state-rpc ={branch = "asset-tx-payment-from-cumulus", git = "https://github.com/georgesdib/substrate/" }
+sc-authority-discovery ={branch = "asset-tx-payment-from-cumulus", git = "https://github.com/georgesdib/substrate/" }
+sc-consensus-uncles ={branch = "asset-tx-payment-from-cumulus", git = "https://github.com/georgesdib/substrate/" }
+try-runtime-cli ={branch = "asset-tx-payment-from-cumulus", git = "https://github.com/georgesdib/substrate/" }
+remote-externalities ={branch = "asset-tx-payment-from-cumulus", git = "https://github.com/georgesdib/substrate/" }
+sc-service-test ={branch = "asset-tx-payment-from-cumulus", git = "https://github.com/georgesdib/substrate/" }
+substrate-frame-cli ={branch = "asset-tx-payment-from-cumulus", git = "https://github.com/georgesdib/substrate/" }
+test-runner-example ={branch = "asset-tx-payment-from-cumulus", git = "https://github.com/georgesdib/substrate/" }
+sc-consensus-manual-seal ={branch = "asset-tx-payment-from-cumulus", git = "https://github.com/georgesdib/substrate/" }
+test-runner ={branch = "asset-tx-payment-from-cumulus", git = "https://github.com/georgesdib/substrate/" }
+chain-spec-builder ={branch = "asset-tx-payment-from-cumulus", git = "https://github.com/georgesdib/substrate/" }
+subkey ={branch = "asset-tx-payment-from-cumulus", git = "https://github.com/georgesdib/substrate/" }
+beefy-gadget ={branch = "asset-tx-payment-from-cumulus", git = "https://github.com/georgesdib/substrate/" }
```

Comment:
based on 
```
error: failed to parse manifest at `/builds/parity/substrate/cumulus/Cargo.toml`
Caused by:
  dependency (beefy-gadget) specification is ambiguous. Only one of `git` or `path` is allowed.
```
from here https://gitlab.parity.io/parity/substrate/-/jobs/1214029
Your patching is not quite correct.

Reply:
mmh, should be correct, though
@joao-paulo-parity Any ideas what might be going wrong with the patching?

Reply:
Is it not because it double patches? As in, I have a patch, and then the bot would also run diener and tries to patch again and has a conflict? But this is me guessing.

Reply:
That makes sense.